helm_deploy/values-dev.yaml

```diff
@@ -122,6 +122,8 @@ generic-prometheus-alerts:
     coreperson:
       - "syscon-devs-dev-migration_coreperson_queue"
       - "syscon-devs-dev-migration_coreperson_dlq"
-
```

Comment:
for some reason I have removed blank lines which make it more readable.  Will update when add sync queues